jvb/src/main/resources/reference.conf

```diff
@@ -29,6 +29,9 @@ videobridge {
     # How long an endpoint can be 'inactive' before it will
     # be considered disconnected
     max-inactivity-limit=3 seconds
+
+    # How often we check endpoint's connectivity status
+    check-interval=500 milliseconds
```

Comment:
is this where the default value comes from?

Reply:
Yeah